ProcessLib/SmallDeformation/SmallDeformationFEM.h

```diff
@@ -474,6 +475,9 @@ class SmallDeformationLocalAssembler
     SecondaryData<typename ShapeMatrices::ShapeType> _secondary_data;
     bool const _is_axially_symmetric;
 
+    /// Flag to indicate whether the local assembly is skipped or not.
+    bool const _is_deactivated;
```

Comment:
Hm... What if, in the future, the set of deactivated subdomains changes over time?

Reply:
Will consider it later.